controllers/etcdcluster_controller.go

```diff
@@ -1404,6 +1404,7 @@ func (r *EtcdClusterReconciler) defragCronJob(log logr.Logger, cluster *etcdv1al
 		log.Error(err, "Unable to connect to etcd")
 		return
 	}
+	defer c.Close()
```

Comment:
This should still leak if there is an error during `Close()`, but that's why we love Golang or not ;).